docs/guides/rate-limits-project-based.mdx

```diff
@@ -0,0 +1,100 @@
+---
+id: project-based-rate-limits
+title: Project-Based Rate Limits
+sidebar_label: Project-Based Rate Limits
+---
+
+## Introduction
+
+This page provides a comprehensive explanation of Ory's Project-Based Rate Limiting. These limits are designed to ensure fair
+resource allocation and maintain the stability and performance of the Ory Network across different project plans and environments.
+
+## What are Project-Based Rate Limits?
+
+Project-Based Rate Limits are controls applied to your entire Ory project, governing the overall request volumes across various
+API endpoints. These limits vary based on your project's plan (Developer, Production, Growth, or Enterprise) and the specific
+environment (Production, Staging, or Workspace).
+
+## How Project-Based Rate Limits Work
+
+Project-Based Rate Limits analyze incoming requests across your entire project and consider factors such as:
+
+- **Project Plan:** Different plans (Developer, Production, Growth, Enterprise) have different rate limit thresholds.
+- **Environment:** Limits may vary between Production, Staging, and Workspace environments.
```

Comment:
This is not fully true, the difference between development and stage is mostly in features, and not rates. We had situations where clients would change their projects from dev to stage in hopes to increase their limits which caused confusion.

Reply:
```suggestion
- **Environment:** Limits are low for Development and Staging environments, and high for Production environments.
```

maybe this? 
what is a workspace env? does that mean the difference of rate limit on workspace/project level?